remote-access/ip-address.md

```diff
@@ -79,6 +79,52 @@ Nmap done: 256 IP addresses (4 hosts up) scanned in 2.41 seconds
 
 Here you can see a device with hostname `raspberrypi` has IP address `192.168.1.8`.
 
+### Getting IPv6 addresses by pinging from a second device (linux device or raspberry pi)
+
+find your own IP by `hostname -I`
```

Comment:
Might be worth clarifying that this is the "own IP" of the "second device" ? :man_shrugging:

Reply:
your are right, I try to follow the nmap example